.github/workflows/python-ci.yml

```diff
@@ -69,9 +69,6 @@ jobs:
           access_token: ${{ github.token }}
       - name: Repo Checkout
         uses: actions/checkout@v2
-        with:
-          ref: ${{ github.event.pull_request.head.ref }}
-          repository: ${{ github.event.pull_request.head.repo.full_name }}
```

Comment:
These settings were required to checkout forked PR's code because the default for `pull_request_target` was the base repo master HEAD. Now that we're using `pull_request` whose default is the branch's head we don't need it.